lib/webpack.config.js

```diff
@@ -8,12 +8,17 @@ const webpack = require('webpack')
 const merge = require('webpack-merge')
 
 const MiniCssExtractPlugin = require('mini-css-extract-plugin')
-const HtmlWebpackPlugin = require('html-webpack-plugin')
```

Comment:
Are we not using this anymore?

Reply:
It's usage is commented out for performance reasons, so I commented out the import as well.

Reply:
Reverted, this was a mistake.